### ISSUE TITLE
Cannot find module 'buffer'

### ISSUE THREAD

**Issue by sido420** [primus/primus#706]
**Version:**
"primus": "7.3.1"
"engine.io": "^3.2.1",
"engine.io-client": "^3.2.1",

**Environment:**
  - **Operating system:** Linux (server) / Mac client
  - **Browser:** Chrome
  - **Node.js:** 10.x

**Expected result:**
Client should load without any issues

**Actual result:**
Getting the following error in browser:
```
Uncaught Error: Cannot find module 'buffer'
    at a (primus.min.js:74)
    at primus.min.js:74
    at Object.18.buffer (primus.min.js:142)
    at a (primus.min.js:74)
    at primus.min.js:74
    at Object.14../keys (primus.min.js:126)
    at a (primus.min.js:74)
    at primus.min.js:74
    at Object.2.component-emitter (primus.min.js:93)
    at a (primus.min.js:74)
```

at the following lines:
```
                        if (!k[g]) {
                            var b = "function" == typeof require && require;
                            if (!l && b)
                                return b(g, !0);
                            if (f)
                                return f(g, !0);
                            l = Error("Cannot find module '" + g + "'");
                            throw l.code = "MODULE_NOT_FOUND",
                            l;
                        }
```

**Steps to reproduce:**
Not sure, but I recently upgraded to the latest primus and engine io etc. Perhaps, a similar cause as https://github.com/primus/primus/issues/519

**Comment by sido420** [comment]
Also, getting this error in Chrome's console:

```
Error: Missing required `engine.io-client` module. Please run `npm install --save engine.io-client`
```

**Comment by sido420** [comment]
Goes away with primus@7.2.3

**Comment by lpinca** [comment]
I suspect this commit is the culprit https://github.com/primus/primus/commit/e3a409452599b21b175346e0be6e7b81975454a5. I will investigate when I get some time.

**Comment by lpinca** [comment]
Should be fixed in primus@7.3.2. Please comment back if this is not the case.

**Comment by lpinca** [comment]
Thanks for reporting the issue.
